### PR TITLE
Bump jinja and opentelemetry versions for airflow support

### PR DESCRIPTION
**Pull request by svdimchenko**
While trying to install soda-core into airflow 2.4.1 I get following conflicts which the PR fixes.
Jinja2:
```
  Because flask (2.2.1) depends on Jinja2 (>=3.0)
   and flask (2.2.0) depends on Jinja2 (>=3.0), flask (2.2.0 || 2.2.1) requires Jinja2 (>=3.0).
  And because flask (2.2.2) depends on Jinja2 (>=3.0)
   and soda-core (3.0.11) depends on Jinja2 (>=2.11,<3.0), flask (2.2.0 || 2.2.1 || 2.2.2) is incompatible with soda-core (3.0.11).
  And because soda-core-bigquery (3.0.11) depends on soda-core (3.0.11)
   and no versions of soda-core-bigquery match >3.0.11,<4.0.0, flask (2.2.0 || 2.2.1 || 2.2.2) is incompatible with soda-core-bigquery (>=3.0.11,<4.0.0).
  And because no versions of flask match >2.2,<2.2.1 || >2.2.1,<2.2.2 || >2.2.2,<2.3
   and apache-airflow (2.4.1) depends on flask (>=2.2,<2.3), soda-core-bigquery (>=3.0.11,<4.0.0) is incompatible with apache-airflow (2.4.1).
  So, because airflow-pm depends on both apache-airflow (2.4.1) and soda-core-bigquery (^3.0.11), version solving failed.
```

tableauserverclient
```
  And because soda-core (3.0.11) depends on requests (>=2.27.1,<2.28.0), soda-core-postgres (>=3.0.11,<4.0.0) requires requests (>=2.27.1,<2.28.0).
  And because tableauserverclient (0.23) depends on requests (>=2.28)
```

### CODE REVIEW

**Comment by vijaykiran** [comment]
@svdimchenko  can you also commit your `dev-requirements.in` file?

**Comment by svdimchenko** [comment]
@vijaykiran I've not made any changes in `dev-requirements.in`, just have updated versions using `pip-compile`

**Comment by vijaykiran** [comment]
@svdimchenko  thank you - I'll test it this week, because we want to bump dbt version as well.

**Comment by svdimchenko** [comment]
@vijaykiran hi, are there any updates regarding the MR review ?

File: soda/core/tests/unit/test_telemetry.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def mock_fn():
     dict_has_keys(span["attributes"], ["user_cookie_id"])
     dict_has_keys(span["context"], ["span_id", "trace_id", "trace_state"])
     dict_has_keys(
-        span["resource"],
+        span["resource"]["attributes"],
```
Comment (vijaykiran) [inline soda/core/tests/unit/test_telemetry.py:38]: This needs changes to our collector, I'll open another PR with all the changes and merge when the collector is updated.

**Comment by vijaykiran** [comment]
Jinja version is bumped, https://github.com/sodadata/soda-core/pull/1728 This should at least fix the airflow compatibility.
for telemetry - we can't bump the version because the payload schema in 1.12 is different than 1.11, which will break collector process.

**Comment by vijaykiran** [comment]
I've an internal issue open for upgrading telemetry API version, closing this because airflow compatibility should work, please open an issue if that's not the case.

**Pull request closed without merging**